python-sdk/src/astro/databases/base.py

```diff
@@ -51,6 +51,8 @@ class BaseDatabase(ABC):
     illegal_column_name_chars_replacement: list[str] = []
     NATIVE_LOAD_EXCEPTIONS: Any = DatabaseCustomError
     DEFAULT_SCHEMA = SCHEMA
+    AUTODETECT_SCHEMA_SUPPORTED: set[Any] = set()
```

Comment:
@sunank200 I think it would be helpful if we add a comment about the values we expect in set.

Reply:
Also, is this ticket applies here too? 

https://github.com/astronomer/astro-sdk/issues/853